boards/intel/ish/intel_ish_5_8_0.yaml

```diff
@@ -5,6 +5,8 @@ arch: x86
 toolchain:
   - zephyr
 ram: 640
+simulation: simics
```

Comment:
and this board definition will be for simulation only, right ?

Reply:
no

Reply:
> no, the twister property is only about twister, .... We might want to add some tweaks to twister to better support this usecase, but this will come in its own change.

then twister will record results using the same testsuite `platform` property name for both HW and simulated execution.

Reply:
then you will have to decide to either run this on hardware or on simulator, there is no need to run this on both. This is a platform simulator.

Reply:
> Twister needs to add `testinstance.platform.simulation` attribute to the json report, at least.

yeah, that can be done. This is what I was referring to in the needed tweaks. This is not something new, we have boards that support multiple simulators and can also run on hardware, all using the same board config.